public/dorequest.php

```diff
@@ -57,19 +57,23 @@ function DoRequestError( $errorMsg )
     case "postactivity":
     case "removecomment":
     case "richpresencepatch":
-    case "submitcodenote":
     case "submitgametitle":
     case "submitlbentry":
     case "unlocks":
     case "uploadachievement":
         $credentialsOK = $validLogin && ( $permissions >= \RA\Permissions::Registered );
         break;
 
-    //	Developer status required:
+     //  Jr Developer status required:
+    case "submitcodenote":
```

Comment:
if we're going to restrict adding code notes to JrDevelopers, we should also restrict `uploadachievement`

Reply:
Changed to make that a Jr. Dev permission.